src/new-client/src/App/Analytics/AnalyticsCore.tsx

```diff
@@ -21,23 +23,25 @@ const Analytics: React.FC = ({ children }) => {
   return (
     <Subscribe source$={analytics$} fallback={children}>
       <SuspenseOnStaleData />
-      <AnalyticsInnerWrapper>
-        <AnalyticsHeader>
-          Analytics
-          <RightNav>
-            {supportsTearOut && <TearOutComponent section="analytics" />}
-          </RightNav>
-        </AnalyticsHeader>
-        <AnalyticsStyle
-          role="region"
-          aria-label="Trade and position analytics"
-          data-qa="analytics__analytics-content"
-        >
-          <ProfitAndLoss />
-          <Positions />
-          <PnL />
-        </AnalyticsStyle>
-      </AnalyticsInnerWrapper>
+      <DraggableTearOut section="analytics">
```

Comment:
Another thought - Since these sections are only tearout'able from the web context, maybe we should wrap them in `DraggableSectionTearOut` in `web/MainRoute.tsx`.
That would keep the section components free from any draggable parent component which isn't needed when in openfin (or when rendering standalone in styleguide/storybook).

Reply:
fixed at `MainRoute.tsx`